app.js

```diff
@@ -39,10 +39,17 @@ addMushroomButton.addEventListener('click', () => {
 });
 
 addFriendButton.addEventListener('click', () => {
-
+    const name = friendInputEl.value;
+    const newFriend = {
+        name: name || `Friend #${Math.floor(Math.random() * 0)}`,
```

Comment:
A random name isn't being created, because you're multiplying by 0, so the random number will always be 0.